src/posts/2021-06-14-Everyday_Accessibility.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: Everyday Accessibility
+description: "Make your social media, digital document, and multimedia content more accessible with these quick and practical steps."
+category: Quick tip
+author: Michele A. Williams, PhD
+date: 2021-06-15
+eleventyNavigation:
+  key: { { category } }
+  title: { { title } }
+tags: quicktip
+---
+
+Even with the best of intentions, you may be making decisions that exclude disabled users. Below are practical, intentional steps you can start today to create and share more accessible social media, digital, and multimedia content.
+
+## Social Media Posts
+
+### Use capital letters in hashtags
+Punctuation helps text-to-speech tools like screen readers speak more accurately, and helps with readability overall. This includes in hashtags, where capital letters help distinguish words.
```

Comment:
```suggestion
### Use capital letters in hashtags

Punctuation helps text-to-speech tools like screen readers speak more accurately, and helps with readability overall. This includes in hashtags, where capital letters help distinguish words.
```

Adding an empty space between the heading and the following sentence so the heading will render properly.

Reply:
I'll be sure to catch these throughout; something to note for content authors. In my writing software (Ulysses) the extra spaces aren't needed when the content is rendered (in the templates they provide) so I didn't know to do it until later.